.github/workflows/cpp-linter.yml

```diff
@@ -0,0 +1,51 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+name: C++ Linter
+
+on:
+  pull_request:
+    paths-ignore:
+      - '.github/**'
+      - 'ci/**'
+      - 'cmake_modules/**'
+    branches:
+      - main
+
+jobs:
+  cpp-linter:
+    runs-on: ubuntu-24.04
+    steps:
+      - uses: actions/checkout@v4
+      - uses: cpp-linter/cpp-linter-action@v2.13.3
```

Comment:
```suggestion
      - uses: cpp-linter/cpp-linter-action@v2.13.4
```

Reply:
I'm not sure if this version is whitelisted. Let's see.